setup.cfg

```diff
@@ -71,6 +71,7 @@ test =
     mock
     pytest
     pytest-cov
+    requests ~= 2.20
```

Comment:
why "pinning" of the release is necessary?  Whenever possible I really prefer to avoid pinning (exact or to a series) so we could react to changes in the dependencies etc, and also to not complicate packaging for distributions (like Debian) where only a single (different) version could be available ATM

Reply:
A lower version bound of 2.20 is recommended for `requests` because previous versions had security issues.  I personally prefer also using an upper version bound for any dependency that uses or approximates semantic versioning, as that helps to prevent one's project from being installed alongside a newer version of a dependency with which it is incompatible due to API changes in a major version.

Reply:
lower bound -- agreed 100%.  Upper bound - I can live with in this particular case, although due to our really basic use of `requests` I do not expect requests 3 to break our code. Let's proceed.